components/runtime/smalltalk/stack-spur/test/results-all.master

```diff
@@ -0,0 +1,24 @@
+SUnit Results
+Squeak5.3
+solaris2.11
+Open Smalltalk Stack VM [StackInterpreterPrimitives VMMaker.oscog-eem.3254]
+4735 run in 0:00:07:34.717, 4676 passes, 42 expected failures, 4 failures, 13 errors, 0 unexpected passes
+Failed Tests
```

Comment:
You should filter out timings (and dates if available) fom the expected test results by means of appropriate COMPONENT_TEST_TRANSFORMS.

Reply:
Yes, exactly.  And in ideal world all tests should pass.